packages/core-data/src/selectors.ts

```diff
@@ -643,6 +711,14 @@ export const getEntityRecordsTotalPages = (
 	return Math.ceil( totalItems / query.per_page );
 };
 
+getEntityRecordsTotalPages.__unstableNormalizeArgs = (
+	args: GetEntityRecordsArgs
+): GetEntityRecordsArgs =>
+	normalizeDeprecatedEntityArgs( args, 'getEntityRecordsTotalPages', {
+		kindArg: 0,
+		nameArg: 1,
```

Comment:
Maybe we don't need to duplicate this across all selectors. 🤔 

I quite like it being explicit, as it makes it easier to spot a potential issue if the args are in a different position, but every single usage is the same at the moment. 😄

Reply:
What if there are new named args in the future, would there have to be a new property? `newArg: 2` or something?

I know it's a draft, but maybe some unit tests might help answer whether the approach is sound.

Reply:
> What if there are new named args in the future

I think JS doesn't support named parameters.

If we decided to change the argument order for a stable selector it'd be a breaking change, but it'd also be possible to update the indices. 

If you mean passing an object with `kind` and `name` properties, then this approach wouldn't work, I'd have to find another way. Possibly it becomes a `path` instead of an index.

I'd like to hear possible objections to the general approach before writing more code.

Reply:
I was thinking more along the lines of any future methods have have params other than `kind` and `type` that don't translate to `kindArg` or `nameArg`. Let's call it `futureResolver( 'tony', 'gary' )`.

Never mind, I was just musing.

Reply:
> I was thinking more along the lines of any future methods have have params other than kind and name that don't translate to kindArg or nameArg. Let's call it futureResolver( 'tony', 'gary' ).

I think I'm only really worried about code that has `kind` and `name` as that's how entities are identified.